src/Sulu/Bundle/MediaBundle/Media/FormatManager/FormatManager.php

```diff
@@ -527,6 +542,7 @@ protected function getMediaData($media)
                     $fileName = $fileVersion->getName();
                     $storageOptions = $fileVersion->getStorageOptions();
```

Comment:
I would also say that `json_decode` should be part of the persisting layer, in all other layers this just causes confusion. I think I would make use of the [`json_array` type](http://doctrine-orm.readthedocs.org/projects/doctrine-dbal/en/latest/reference/types.html#json-array) of doctrine for that.